wire-ios-data-model/Tests/MLS/MockMLSService.swift

```diff
@@ -172,4 +172,11 @@ class MockMLSService: MLSServiceInterface {
         calls.scheduleCommitPendingProposals.append((groupID, commitDate))
     }
 
+    func createOrJoinSubgroup(
+        parentQualifiedID: QualifiedID,
+        parentID: MLSGroupID
+    ) async {
+
```

Comment:
finish mock implementation or fatalError()

Reply:
I want to generate this mock and reuse it but I thought to do it in another PR.